Islandora/islandora_solr_search
# Add paged Atom feed option to search results

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2440)

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

https://groups.google.com/d/topic/islandora/8eZCuy2NCCc/discussion

# What does this Pull Request do?

Provides a new Secondary Display Profile option: Atom feed, an alternative to RSS with more features (especially paged results). 

# What's new?

The new Atom option rewrites the RSS output to make it compatible with Atom feed standards, and is offered as a new display profile. The Atom feed is paged and so can provide more results than the current Solr limit.

# How should this be tested?

* Open the Solr configuration page
* Check off the Atom feed option
* Run a search
* Click the button and view the Atom results
* Try the paged link options to confirm that paging works too

# Additional Notes:

Currently using the RSS icon for the feed, which may be confusing if you're using both Atom and RSS -- although you shouldn't really be using both at the same time, should you? Open to alternative icons.

# Interested parties
@Islandora/7-x-1-x-committers
